metricbeat/docs/modules/azure/app_state.asciidoc

```diff
@@ -0,0 +1,25 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-metricset-azure-app_state]]
+[role="xpack"]
+=== Azure app_state metricset
+
+beta[]
+
+include::../../../../x-pack/metricbeat/module/azure/app_state/_meta/docs.asciidoc[]
+
+This is a default metricset. If the host module is unconfigured, this metricset is enabled by default.
```

Comment:
Whats host module here?

Reply:
fixed, it was set as default in the manifest file